### PR TITLE
remove unused functions and variables

### PR DESCRIPTION
**Pull request by surajssd**
Lot of functions and variables were unused, so this commit removes all those functions.

### CODE REVIEW

**Comment by surajssd** [comment]
If there are things that I should be taking care of please suggest! Also being new to this project, this is my first PR.

File: cmd/kpod/parse.go [hunk 1]
```diff
@@ -5,79 +5,22 @@ package main
 
 import (
```
Comment (mheon) [inline cmd/kpod/parse.go]: Leave the things in this file - we're slowly adding functionality to kpod run which requires them.

File: cmd/kpod/create.go [hunk 2]
```diff
@@ -12,7 +12,6 @@ import (
 	"github.com/pkg/errors"
 	"github.com/projectatomic/libpod/libpod"
 	"github.com/urfave/cli"
```
Comment (mheon) [inline cmd/kpod/create.go]: Leave things in this file - they're not used yet, but we're slowly writing code to wire them in

File: cmd/kpod/ps.go [hunk 3]
```diff
@@ -31,7 +31,6 @@ type psOptions struct {
 	noTrunc   bool
 	quiet     bool
 	size      bool
-	label     string
```
Comment (mheon) [inline cmd/kpod/ps.go]: @baude Should we keep this? Not sure

File: cmd/kpod/parse.go (hunk 1 above)
Comment (surajssd) [reply cmd/kpod/parse.go]: done!

File: cmd/kpod/create.go (hunk 2 above)
Comment (surajssd) [reply cmd/kpod/create.go]: done!

File: cmd/kpod/ps.go (hunk 3 above)
Comment (TomSweeneyRedHat) [reply cmd/kpod/ps.go]: FWIW, I'm not seeing it being used anywhere.  Looks to be safe to remove for now and readd if necessary.   @baude WDYT?

File: libpod/common/common.go [hunk 4]
```diff
@@ -69,15 +59,6 @@ func IsValidBool(str string) bool {
...
 
-// GetPolicyContext creates a signature policy context for the given signature policy path
-func GetPolicyContext(path string) (*signature.PolicyContext, error) {
-	policy, err := signature.DefaultPolicy(&types.SystemContext{SignaturePolicyPath: path})
-	if err != nil {
-		return nil, err
-	}
-	return signature.NewPolicyContext(policy)
-}
-
```
Comment (TomSweeneyRedHat) [inline libpod/common/common.go]: These two functions may not be used ATM, but I"m always hesitant to remove tool like functions that someone found useful at some point in time.

File: cmd/kpod/ps.go (hunk 3 above)
Comment (baude) [reply cmd/kpod/ps.go]: I would say to leave it for now, just me though

File: libpod/common/common.go [hunk 5]
```diff
@@ -45,15 +44,6 @@ func GetSystemContext(signaturePolicyPath, authFilePath string) *types.SystemCon
...
 
-// CopyStringStringMap deep copies a map[string]string and returns the result
-func CopyStringStringMap(m map[string]string) map[string]string {
-	n := map[string]string{}
-	for k, v := range m {
-		n[k] = v
-	}
-	return n
-}
-
```
Comment (baude) [inline libpod/common/common.go]: Rather than delete, please move to the util file.

File: cmd/kpod/ps.go (hunk 3 above)
Comment (surajssd) [reply cmd/kpod/ps.go]: ok done, left it as it is!

File: libpod/common/common.go (hunk 5 above)
Comment (surajssd) [reply libpod/common/common.go]: moved to util

**Comment by surajssd** [comment]
@baude @TomSweeneyRedHat done as you suggested! PTAL1

File: libpod/util.go [hunk 6]
```diff
@@ -42,3 +45,21 @@ func FuncTimer(funcName string) {
...
+}
+
+// GetPolicyContext creates a signature policy context for the given signature policy path
+func GetPolicyContext(path string) (*signature.PolicyContext, error) {
+	policy, err := signature.DefaultPolicy(&types.SystemContext{SignaturePolicyPath: path})
+	if err != nil {
+		return nil, err
+	}
+	return signature.NewPolicyContext(policy)
+}
```
Comment (TomSweeneyRedHat) [inline libpod/util.go]: Thanks for moving these!

**Comment by TomSweeneyRedHat** [comment]
LGTM, thanks for the PR @surajssd

**Comment by rhatdan** [comment]
@surajssd Could you rebase.

**Comment by surajssd** [comment]
@rhatdan done!

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**